### PR TITLE
en: update TLS certificate for the admission controller

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Update Webhook TLS configuration document.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/171
- Other reference link(s):

### CODE REVIEW

File: en/enable-admission-webhook.md [hunk 1]
```diff
@@ -21,6 +27,22 @@ With a default installation, TiDB Operator disables the admission controller. Ta
       create: true
     ```
 
+    * If your Kubernetes cluster version >= v1.13.0, you can enable the Webhook feature by the configuration above.
```
Comment (lilin90) [inline en/enable-admission-webhook.md:30]:
```suggestion
    * If your Kubernetes cluster version >= v1.13.0, enable the Webhook feature by using the configuration above.
```

File: en/enable-admission-webhook.md [hunk 2]
```diff
@@ -21,6 +27,22 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+
+    * If your Kubernetes cluster version < v1.13.0, you need to run the following command and configure the `admissionWebhook.cabundle` in `values.yaml` as the return value:
```
Comment (lilin90) [inline en/enable-admission-webhook.md:32]:
```suggestion
    * If your Kubernetes cluster version < v1.13.0, run the following command and configure the `admissionWebhook.cabundle` in `values.yaml` as the return value:
```

File: en/enable-admission-webhook.md [hunk 3]
```diff
@@ -21,6 +27,22 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl get configmap -n kube-system extension-apiserver-authentication -o=jsonpath='{.data.client-ca-file}' | base64 | tr -d '\n'
+        ```
+
+        ```yaml
+        admissionWebhook:
+          # Configure `admissionWebhook.cabundle` as the return value of the command above
```
Comment (lilin90) [inline en/enable-admission-webhook.md:42]:
```suggestion
          # Configure the value of `admissionWebhook.cabundle` as the return value of the command above
```

File: en/enable-admission-webhook.md [hunk 4]
```diff
@@ -47,6 +69,128 @@ With a default installation, TiDB Operator disables the admission controller. Ta
 
     To install or update TiDB Operator, see [Deploy TiDB Operator in Kubernetes](deploy-tidb-operator.md).
 
+## Set TLS certificate for the admission controller
```
Comment (lilin90) [inline en/enable-admission-webhook.md:72]:
```suggestion
## Set the TLS certificate for the admission controller
```

File: en/enable-admission-webhook.md [hunk 5]
```diff
@@ -47,6 +69,128 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+
+By default, the admission controller and Kubernetes api-server skip the [TLS verification](https://kubernetes.io/docs/tasks/access-kubernetes-api/configure-aggregation-layer/#contacting-the-extension-apiserver). To manually enable and configure the TLS verification between the admission controller and Kubernetes api-server, take the following steps:
+
+1. Generate the custom certificate
```
Comment (lilin90) [inline en/enable-admission-webhook.md:76]:
```suggestion
1. Generate the custom certificate.
```

File: en/enable-admission-webhook.md [hunk 6]
```diff
@@ -47,6 +69,128 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+    }
+    ```
+
+    After executing Step 4, run the `ls` command. The following files should be listed in the `cfssl` folder:
+
+    ```bash
+    ca-config.json    ca-csr.json    ca-key.pem    ca.csr    ca.pem
+    ```
+
+2. Generate the certificate for the admission controller
```
Comment (lilin90) [inline en/enable-admission-webhook.md:108]:
```suggestion
2. Generate the certificate for the admission controller.
```

File: en/enable-admission-webhook.md [hunk 7]
```diff
@@ -47,6 +69,128 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+
+    First, create the default `webhook-server.json` file:
```
Comment (lilin90) [inline en/enable-admission-webhook.md:110]: Suggest using a sub-step ordered list for this section.
```suggestion
    1. Create the default `webhook-server.json` file:
```

File: en/enable-admission-webhook.md [hunk 8]
```diff
@@ -47,6 +69,128 @@ With a default installation, TiDB Operator disables the admission controller. Ta
...
+
+3. Create a secret in the Kubernetes cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic <secret-name> --namespace=<namespace> --from-file=tls.crt=~/cfssl/webhook-server.pem --from-file=tls.key=~/cfssl/webhook-server-key.pem --from-file=ca.crt=~/cfssl/ca.pem
+    ```
+
+4. Modify `values.yaml` and install/upgrade TiDB Operator
```
Comment (lilin90) [inline en/enable-admission-webhook.md:172]:
```suggestion
4. Modify `values.yaml`, and install or upgrade TiDB Operator.
```

**Comment by ran-huang** [comment]
@lilin90 All comments addressed. PTAL~

**Pull request merged**